scripts/ws_client.py

```diff
@@ -101,7 +101,7 @@ def json_deserialize(message):
     :param message: The message to deserialize
     """
     def json_to_dataframe(data: str) -> pandas.DataFrame:
-        dataframe = pandas.read_json(data, orient='split')
+        dataframe = pandas.DataFrame.from_dict(data, orient='tight')
```

Comment:
it's now no longer leaking, which is great.

What i don't like are 2 things:
* it's a **breaking change** - as the transferred message slightly changed - which means once a producer updates to this - all consumers need to follow suite immediately, or will not be able to receive any more messages. I think we can counter this quite easily in the producer by running something around `str(orjson.dumps(df.to_dict(orient='')))` - which will revert it back to a string.
* message size almost doubles (in my 1000 columns test, it jumps from 13Mb to >19mb)

a quick output of the prior / after 

```
dumps: {"index":[0],"columns":["date","open","high","low","close","volume"],"data":[[1593907200000,20.993428306022466,22.855164004851282,18.345739767921774,22.516458162430492,199.76584662527665]]}
to_json: {"columns":["date","open","high","low","close","volume"],"index":[0],"data":[[1593907200000,20.993428306,22.8551640049,18.3457397679,22.5164581624,199.7658466253]]}
```

The point that changes is the precision in numbers.
I don't think we can (or should) counter this - it's more an awareness thing - not something "really" problematic.
The reason for this is that apparently, `to_json()` is doing some [funky data conversion](https://stackoverflow.com/questions/62927218/pandas-dataframe-with-float64-incorrectly-exported-by-to-json-function) - so the prior version was slightly changing decimals between producer and consumer - while the new mode won't.